corehq/apps/export/models/new.py

```diff
@@ -920,6 +933,53 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+    name = "Messages"
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        columns = [
+            ("Contact Type", "couch_recipient_doc_type", DOC_TYPE_TRANSFORM),
+            ("Contact ID", "couch_recipient", None),
+            ("Timestamp", "date", None),
+            ("Owner Name", "couch_recipient", OWNER_ID_TRANSFORM),
+            ("Phone Number", "phone_number", None),
+            ("Direction", "direction", None),
+            ("Message", "text", None),
+            ("Type", "workflow", WORKFLOW_TRANSFORM),
+        ]
+        main_table = TableConfiguration(
+            label='Messages',
+            path=MAIN_TABLE,
+            selected=True,
+            columns=[
+                ExportColumn(
+                    label=col[0],
+                    item=ExportItem(
+                        path=[PathNode(name=col[1])],
+                        transform=col[2]
+                    ),
+                    selected=True
+                )
+                for col in columns
+            ],
+        )
+        if schema.include_metadata:
+            main_table.columns.append(ExportColumn(
+                label="Message Log ID",
+                item=ExportItem(
+                    path=[PathNode(name='_id')]
+                ),
+                selected=True,
+            ))
+
+        instance = cls(domain=schema.domain, tables=[main_table])
+        cls._insert_system_properties(instance.domain, schema.type, instance.tables[0])
```

Comment:
Looks like that doesn't happen because I'm creating the schema without any `group_schemas` since all properties are system properties. Do you think it's worth moving those properties over to a `group_schema` in `SMSExportDataSchema`?